README.md

```diff
@@ -105,36 +87,38 @@ After that, you can run tests faster by adding the `--load-env` flag along with
 the received value. This way you won't waste time redeploy the environment.
 For example:
 ```shell
-pytest --alluredir my-allure-234 -s pytest_tests/tests/services/rest_gate/test_rest_bearer.py --load-env env_files/persisted_env_awxyrbxdwu 
+pytest -s pytest_tests/tests/services/rest_gate/test_rest_bearer.py --load-env env_files/persisted_env_awxyrbxdwu 
 ```
-If anything goes wrong, first advice is to check .github/workflows/run-tests.yml, to ensure you've done all required steps.
 
-3. Generate report
+## pytest marks
 
-If you opted to install Allure CLI, you can generate a report using the command `allure generate`. The web representation of the report will be under `allure-report` directory:
-```shell
-$ allure generate my-allure-123
-$ ls allure-report/
-app.js  data  export  favicon.ico  history  index.html  plugins  styles.css  widgets
-```
+Custom pytest marks used in tests:
+* `sanity` - a short subset of tests to ensure basic NeoFS functionality works.
 
-To inspect the report in a browser, run
-```shell
-$ allure serve my-allure-123
-```
+## Tests/Libraries structure
+
+Tests are located under `pytest_tests/tests`.
+
+Different libraries used by tests are located under `pytest_tests/lib/` and `neofs_testlib/`.
+
+At this moment, there is no logic behind libraries location. 
```

Comment:
:tada:

Reply:
We can improve it, but not in the near future, it doesn't really affect anything.